internal/nodes/avi_model_evh_nodes.go

```diff
@@ -1842,12 +1871,16 @@ func RemoveFqdnFromEVHVIP(vsNode *AviEvhVsNode, hostsToRemove []string, key stri
 		}
 	}
 }
-func (o *AviObjectGraph) RemoveHTTPRefsFromEvh(httpPol, hppmapName string, evhNode *AviEvhVsNode) {
-
+func (o *AviObjectGraph) RemoveHTTPRefsStringGroupsFromEvh(httpPol, hppmapName string, evhNode *AviEvhVsNode) {
+	var stringGroupToRemove []string
 	for i, pol := range evhNode.HttpPolicyRefs {
 		if pol.Name == httpPol {
 			for j, hppmap := range evhNode.HttpPolicyRefs[i].HppMap {
 				if hppmap.Name == hppmapName {
+					if len(evhNode.HttpPolicyRefs[i].HppMap[j].StringGroupRefs) > 0 {
+						sgref := strings.Split(evhNode.HttpPolicyRefs[i].HppMap[j].StringGroupRefs[0], "=")[1]
```

Comment:
will this value `evhNode.HttpPolicyRefs[i].HppMap[j]` always have only one StringGroupRefs ?

Reply:
Yes, it will have one StringGroupRef value. I have added the field as type []string mostly to keep it inline with Path field which also has one value but has type []string. See **AviHostPathPortPoolPG** struct definition.